state/indexer/sink/psql/psql.go

```diff
@@ -1,6 +1,8 @@
-// Package psql implements an event sink backed by a PostgreSQL database.
+//nolint:goconst
```

Comment:
I actually went and created the string consts in #1510, those concatenations do useless work repeatedly.
Even better for performance would be to create prepared SQL statements, because these queries are used in a loop.

Reply:
FYI, this code will likely be removed soon, as ADR101 makes it unnecessary to be part of CometBFT

Reply:
I cherry-picked 90fae42c0d into this PR

Reply:
Thanks; sorry the commit description was misleading, I edited it before making more changes.

Reply:
NP, will get squashed :-)